webrender/src/render_backend.rs

```diff
@@ -132,7 +132,7 @@ static NEXT_NAMESPACE_ID: AtomicUsize = ATOMIC_USIZE_INIT;
 /// GPU-friendly work which is then submitted to the renderer in the form of a frame::Frame.
 ///
 /// The render backend operates on its own thread.
-pub struct RenderBackend {
+pub struct RenderBackend<T> {
```

Comment:
Is there a reason this needs to be generic instead of using a trait object? Having it generic is going to increase wrench's already bad build times and we don't have any need to inline or devirtualize the notifier.

Reply:
I have rewritten the changes to retain the trait object.